src/allencell_ml_segmenter/training/view.py

```diff
@@ -296,18 +297,46 @@ def getTypeOfWork(self) -> str:
         return "Training"
 
     def showResults(self) -> None:
-        csv_path: Optional[Path] = (
-            self._experiments_model.get_latest_metrics_csv_path()
+        # double check to see if a ckpt was generated
+        exp_path: Optional[Path] = (
+            self._experiments_model.get_user_experiments_path()
         )
-        if csv_path is None:
-            raise RuntimeError("Cannot get min loss from undefined csv")
-        min_loss: Optional[float] = FileUtils.get_min_loss_from_csv(csv_path)
-        if min_loss is None:
-            raise RuntimeError("Cannot compute min loss")
-        dialog_box = InfoDialogBox(
-            "Training finished -- Final loss: {:.3f}".format(min_loss)
+        if exp_path is None:
+            raise ValueError(
+                "Experiments path should not be None after training complete."
+            )
+        exp_name: Optional[str] = self._experiments_model.get_experiment_name()
```

Comment:
Are these new validation check related to the pr's focus?  Or just something you noticed we should have?

Reply:
This is to satisfy mypy requirements

Reply:
punting to the refactor ticket (discussed in person)

Reply:
https://github.com/AllenCell/allencell-segmenter-ml/issues/522